src/app/src/__tests__/utils.tests.js

```diff
@@ -1445,15 +1445,10 @@ it('creates a URL for POSTing the claim a facility form', () => {
 
 it('checks whether the claim a facility form is valid', () => {
     const validForm = {
-        email: 'email@example.com',
```

Comment:
Why did you remove 'email' field from test as it's still populated in the request body?